web/src/App.tsx

```diff
@@ -140,6 +142,36 @@ const App = () => {
         <div className="wrapper">
             <Header />
             <main className="main grow">
+                <div className="container">
+                    <div className="flex flex-nowrap items-start border border-yellow-500 rounded-lg mt-6 p-5 bg-yellow-50 gap-x-3">
+                        <div className="text-yellow-500">
+                            <FontAwesomeIcon icon={faExclamationTriangle} size="lg" />
+                        </div>
+                        <div>
+                            Due to the high load, some services may not work properly.
```

Comment:
Due to high demand, some services may not function optimally. Check status updates on Twitter.